spec/lib/pre_assembly/digital_object_spec.rb

```diff
@@ -419,100 +500,22 @@ def add_object_files(extension = 'tif', all_files_public: false)
         allow(bc).to receive(:content_md_creation).and_return('filename')
         add_object_files('tif')
         add_object_files('jp2')
+        allow(SecureRandom).to receive(:uuid).and_return('1', '2', '3', '4')
       end
 
-      it 'generates the expected xml text' do
-        expect(noko_doc(object.send(:create_content_metadata, false))).to be_equivalent_to(exp_xml)
+      it 'generates the expected structural' do
+        expect(object.send(:build_structural).to_h).to eq expected
       end
     end
+  end
 
-    describe 'with file attributes' do
-      let(:exp_xml) do
-        noko_doc <<-XML
-        <contentMetadata type="book" objectId="gn330dv6119">
-          <bookData readingOrder="ltr"/>
-          <resource type="page" sequence="1" id="gn330dv6119_1">
-            <label>Page 1</label>
-            <file id="image1.jp2" preserve="yes" shelve="yes" publish="yes">
-              <checksum type="md5">1111</checksum>
-            </file>
-            <file id="image1.tif" preserve="yes" shelve="yes" publish="yes">
-              <checksum type="md5">1111</checksum>
-            </file>
-          </resource>
-          <resource type="page" sequence="2" id="gn330dv6119_2">
-            <label>Page 2</label>
-            <file id="image2.jp2" preserve="yes" shelve="yes" publish="yes">
-              <checksum type="md5">2222</checksum>
-            </file>
-            <file id="image2.tif" preserve="yes" shelve="yes" publish="yes">
-              <checksum type="md5">2222</checksum>
-            </file>
-          </resource>
-        </contentMetadata>
-        XML
-      end
-
-      let(:bc) { create(:batch_context, :public_files, bundle_dir: 'spec/test_data/images_jp2_tif') }
-
-      before do
-        allow(object).to receive(:druid).and_return(druid)
-        allow(object).to receive(:object_type).and_return('')
-        allow(bc).to receive(:content_structure).and_return('simple_book')
-        allow(bc).to receive(:content_md_creation).and_return('filename')
-        add_object_files('tif', all_files_public: true)
-        add_object_files('jp2', all_files_public: true)
-      end
-
-      it 'generates the expected xml text' do
-        expect(object.send(:create_content_metadata, true)).to be_equivalent_to(exp_xml)
-      end
+  describe '#content_md_creation_style' do
```

Comment:
I'm not understand this.

Reply:
This is a DB field, so It would be best if we change it in a separate PR if we want to do so.